src/borrower/deployer.sol

```diff
@@ -103,7 +103,7 @@ contract BorrowerDeployer is FixedPoint {
 
     /// @notice deploys the borrower contracts and wires them together
     /// @param initNAVFeed boolean flag if a NAV feed should be deployed
-    function deploy(bool initNAVFeed) public {
+    function deploy(bool initNAVFeed, bool fileDiscountRate) public {
```

Comment:
Okay, so basically in the PV feed case we pass `ONE` to the borrower deployer as default `discountRate` but we never set it. 

I think this might be okay for now. In the ideal case we would have separate Borrower deployers.